src/pages/HomeScreenOfTheElderly/HomeScreenOfTheElderly.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import * as S from "./HomeScreenOfTheElderlyStyled";
+import {
+  UserOverview, Card, Button, Layout,
+} from "../../components";
+import { userData } from "../../_mock/userData";
+
+
+export const HomeScreenOfTheElderly = () => {
+  return (
+    <Layout hasTabBar>
+
+      <S.ContainerPage>
+
+        <S.ContainerUserOverview>
+          <UserOverview userData={userData} />
+        </S.ContainerUserOverview>
+
+        <S.ContainerOne borderTop="1px solid #D8CDEE" borderBottom="1px solid #D8CDEE">
+
+          <S.Paragraph>Como você prefere pedir ajuda?</S.Paragraph>
+          <S.ContainerCards>
+            <Card
```

Comment:
@Rafaella-Almeida um detalhe, ao clicar nesse card leva para a rota `ask-for-help`. Pode adicionar por gentileza?